cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,15 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/');
+    cy.get(':nth-child(2) > .nav-link').click();
+    cy.url().should('contain', '/login');
 
+    cy.get(':nth-child(1) > .form-control').type('oaao@gmail.com');
+    cy.get(':nth-child(2) > .form-control').type('1234qaz');
+
+    cy.get('.btn').click();
+
+    cy.get('.navbar').should('contain', 'try');
```

Comment:
The assertion to check if the navbar contains 'try' is not clear. It would be better to assert something more specific to the user being logged in, such as a username or logout button.